docs/SOPs/dataset_acknowledgement_SOP.md

```diff
@@ -121,33 +121,18 @@ this file. For example, to run against dev:
-- **sample_type**: Indicate the type of sample (Can choose more than one)
-- **health_status**: Indicate if `normal`, `not normal` or `both`
-- **phenotype**: If the health status is not `normal`, indicate the phenotype here
-- **assay_type**: Indicate the technology used to generate the library preparations. If the technology is not listed, please add it to the dropdown
-- **organ**: Indicate, with ontology terms from UBERON when possible, the organ where the samples come from.
-- **living_eu_donors**: When human, indicate if they were living EU citizens when the samples were collected. When other organism, indicate `no,none`
-- **nucleic_acid_source**: Indicate if bulk, single cell or single nucleus
-- **data_available**: `yes` if the raw sequencing data is available
-- **technical_benchmarking**: Indicate if the dataset is a benchmarking experiment (e.g. how does tissue storage affect library preparation) 
-- **broker_to_archives**: Usually `no` if from archives, `yes` if from contributor and need to broker to ENA
-- **broker_to_scea**: Based on their guidelines, indicate `yes` or `no`
-- **pub_title**: Indicate publication title
-- **hca_pub**: Indicate if it's an HCA publication
-- **pub_link**: Indicate the link to the publication (Can be any link as long as it resolves to the manuscript in the journal)
-- **pmid**: Indicate pmid. An easy way to do it is by using the tracking sheet function `setPmid()` indicating the cell with the title as a parameter
-- **doi**: Indicate DOI. Make sure it starts with `10.`.
+When needed, datasets can also be added manually. First, make sure to check that the dataset is not duplicated by searching in Ingest by DOI or project title. This can be checked too while trying to create a new project based on DOI or GEO accession, as Ingest will search for them in the database to avoid duplication.
+
+When the dataset is not present, please fill in **at least** the following information:
+- **Accessions**: If available, add the corresponding data accessions.
+- **Data access**: Fill in the type of data access (managed, open, mix, it's complicated). 
+- **Organism**: Indicate the organism, generally human.
+- **Organs**: Indicate the organs used in the experiment.
+- **library preparation**: Indicate the technology used to generate the library preparations.
+- **Living eu donors**: When human, indicate if they were living EU citizens when the samples were collected on the admin area, under the notes section) .
+- **Cell count**: Add the number of cells sequenced in the experiment, or at least an approximation
+- **Broker to archives**: If data comes from contributor and need to broker to ENA, add this to the notes section at the admin area.
+- **Broker to scea**: Always check if the dataset is eligible for SCEA based on their guidelines. If it is eligible for SCEA, please indicate this in the notes section of the admin area.
+- **Contributors**: If available, fill in the contributors section
```

Comment:
Not really needed, it's nice but on the discovery/triage phase it's not really necessary and it kinda adds some drag to it

Reply:
We've been speaking about this, and this section is not about triaging but about manually adding a project to ingest, so we are gonna leave it as it is.